packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeFeeRegistry.sol

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./AirnodeFeeRegistryRolesWithManager.sol";
+import "./interfaces/IAirnodeFeeRegistry.sol";
+
+/// @title Contract for storing the price of a airnode endpoints
+/// @notice The AirnodeFeeRegistry contract is a central contract that
+/// all the different AirnodeTokenLock contracts (of different chains)
+/// will be calling to find out what the USD price of a chain-airnode-endpoint is
+contract AirnodeFeeRegistry is
+    AirnodeFeeRegistryRolesWithManager,
+    IAirnodeFeeRegistry
+{
+    /// @notice The USD price of an endpoint for an airnode on a specific chain
+    mapping(uint256 => mapping(address => mapping(bytes32 => uint256)))
+        public chainIdToAirnodeToEndpointToPrice;
+
+    /// @notice The USD price of an endpoint for an airnode across all chains
+    mapping(address => mapping(bytes32 => uint256))
+        public airnodeToEndpointToPrice;
+
+    /// @notice The default price of an endpoint for an airnode on a specific chain
+    mapping(uint256 => mapping(address => uint256))
+        public defaultChainAirnodePrice;
+
+    /// @notice The default price of an endpoint on an airnode
+    mapping(address => uint256) public defaultAirnodePrice;
+
+    /// @notice The default price of an endpoint for a chain
+    mapping(uint256 => uint256) public defaultChainPrice;
+
+    /// @notice The global default price of an endpoint
+    uint256 public defaultPrice;
+
+    /// @notice A flag that indicates which price to default to in
+    /// the getAirnodeEndpointFee function
+    mapping(address => bool) public airnodeEndpointFlag;
+
+    /// @notice Prices will have upto 6 decimal places
+    uint8 public immutable decimals = 6;
```

Comment:
Why is this state var immutable if we are not initializing it in the constructor? Are we planning to do that or should it just be a constant?

Reply:
It was going to be initialized in the constructor but its better to have it be a constant because we don't want it to change in the future